src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -53,6 +55,16 @@ export default function VideoVisitLocation({ appointment, facility }) {
           />
         </div>
       )}
+      {isClinicVideoAppointment(appointment) && (
```

Comment:
I noticed this when testing, the providers were in a weird place between text telling a veteran to go to the below location and the actual location info.